oioioi/amppz/README.rst

```diff
@@ -1 +1 @@
-Handles `Akademickie Mistrzostwa Polski w Programowaniu Zespołowym type <http://amppz.mimuw.edu.pl/>`_ programming contests.
+Handles `Akademickie Mistrzostwa Polski w Programowaniu Zespołowym type <https://pl.wikipedia.org/wiki/Akademickie_Mistrzostwa_Polski_w_Programowaniu_Zespo%C5%82owym>`_ programming contests.
```

Comment:
The current url for AMPPZ is https://amppz.edu.pl

Reply:
It's true. In the past it was also http://amppz.tcs.uj.edu.pl/ and http://amppz.ii.uni.wroc.pl/

Reply:
Maybe it’s best to link to Wikipedia since the link changes every few years?

Reply:
Let's move our discussion to the issue: https://github.com/sio2project/oioioi/issues/201#issuecomment-1813132670 . Please see my full suggestion there.